fdbclient/BackupContainerFileSystem.actor.cpp

```diff
@@ -977,6 +976,8 @@ class BackupContainerFileSystemImpl {
 			// TODO: Reenable the sanity check after TooManyFiles error is resolved
 			if (false && g_network->isSimulated()) {
 				// Sanity check key ranges
+				ASSERT(false);
```

Comment:
Do we want to commit this assert? Seems like some debugging assist you might have added?

Reply:
Right now this codepath won't be executed. But i think we should still keep this assert because now we require any caller that wants to take a snapshot needs to pass a valid db object. So if someone were to enable this codepath we would need to make sure a db object is passed. I can add a comment to make this more clear

Reply:
I would vote for deleting `dead code`; for more understanding any reason we want to keep it around?